js/holmes.js

```diff
@@ -279,9 +282,15 @@
 
       // loop over all the elements
       // in case this should become dynamic, query for the elements here
-      holmes.prototype.regex = new RegExp('(' + holmes.prototype.searchString + ')(?![^<]*>)', 'gi');
+      if (holmes.prototype.options.mark) {
+        holmes.prototype.regex = new RegExp('(' + holmes.prototype.searchString + ')(?![^<]*>)', 'gi');
+      }
```

Comment:
This is a good change. Although both of the operations (a single boolean if and a regex instantiation) aren't too heavy.